openerp/addons/base/tests/test_display_address.py

```diff
@@ -0,0 +1,11 @@
+from openerp.tests import common
+
+
+class TestDisplayAddress(common.TransactionCase):
+    def test_display_address(self):
+        """ _display_address() should not include empty lines when some data is missing """
```

Comment:
What if I used *data* as uncountable? http://www.oxforddictionaries.com/definition/learner/data. Would it help if I changed to *some of the data is missing*?